src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient+Configuration.swift

```diff
@@ -114,7 +114,7 @@ extension HTTPClient {
 				.distribution
 				.appending(
 					"version",
-					apiVersion,
+					"v2",
```

Comment:
why remove the parameter apiVersion with a fixed string?

Reply:
We only have a global apiVersion parameter right now, but in this case only the version for one endpoint changed